e2e/homepage.spec.ts

```diff
@@ -22,3 +22,33 @@ test.describe("Homepage", () => {
     await expect(page.getByText(copyrightText, { exact: true })).toBeVisible();
   });
 });
+
+test.describe("Blog", () => {
```

Comment:
let's move this out of `homepage.spec` since this isn't about the homepage.

Reply:
I'll move this to its own file. And I'll hold off on making any more tests or adjustments to tests afterward. I do want to see your take on how these tests should be done, considering you may have experience working on more back-end systems than me in my time.

Reply:
Done